client/js/render.js

```diff
@@ -0,0 +1,192 @@
+"use strict";
+const $ = require("jquery");
+const templates = require("../views");
+const options = require("./options");
+const utils = require("./utils");
+const sorting = require("./sorting");
+
+const chat = $("#chat");
+const sidebar = $("#sidebar, #footer");
+
+module.exports = {
+	buildChannelMessages,
+	buildChatMessage,
+	renderChannel,
+	renderChannelMessages,
+	renderChannelUsers,
+	renderNetworks
+};
```

Comment:
So, there are 2 standards, 1 is the add it at the end and 1 is the add it at the beginning.

I tend to add it at the beginning because I like have it front and centre, one of the first things you see when you go in, because it gives you a clear indication what is exported straight away.